app/services/bulk_api_import/fhir_patient_importer.rb

```diff
@@ -8,10 +8,12 @@ def initialize(patient_resource)
   end
 
   def import
-    transformed_params = Api::V3::PatientTransformer.from_nested_request(build_attributes)
-    patient = MergePatientService.new(transformed_params, request_metadata: request_metadata).merge
-    AuditLog.merge_log(import_user, patient) if patient.present?
-    patient
+    merge_result = build_attributes
+      .then { Api::V3::PatientTransformer.from_nested_request(_1) }
+      .then { MergePatientService.new(_1, request_metadata: request_metadata).merge }
+
+    AuditLog.merge_log(import_user, merge_result) if merge_result.present?
+    merge_result
```

Comment:
Refactored this to match how I wrote it for the other resources. The chaining version expresses the intent better to my eyes.